Unable to install npm package on Windows

If I run `npm install -g @sourcegraph/src` I get the following output:

```
PS C:\Dev\Sourcegraph\cody> npm install -g @sourcegraph/src
npm ERR! code 1
npm ERR! path C:\Users\danny\AppData\Roaming\npm\node_modules\@sourcegraph\src
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node install.js
npm ERR! Downloading https://github.com/sourcegraph/src-cli/releases/download/5.2.1/src-cli_5.2.1_windows_amd64.tar.gz
npm ERR! node:internal/fs/utils:350
npm ERR!     throw err;
npm ERR!     ^
npm ERR!
npm ERR! Error: ENOENT: no such file or directory, chmod 'src.exe'
npm ERR!     at Object.chmodSync (node:fs:1991:3)
npm ERR!     at Unpack.<anonymous> (C:\Users\danny\AppData\Roaming\npm\node_modules\@sourcegraph\src\install.js:60:36)
npm ERR!     at Unpack.emit (node:events:517:28)
npm ERR!     at C:\Users\danny\AppData\Roaming\npm\node_modules\@sourcegraph\src\node_modules\tar\lib\parse.js:235:25
npm ERR!     at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
npm ERR!   errno: -4058,
npm ERR!   syscall: 'chmod',
npm ERR!   code: 'ENOENT',
npm ERR!   path: 'src.exe'
npm ERR! }
npm ERR!
npm ERR! Node.js v18.18.2

npm ERR! A complete log of this run can be found in: C:\Users\danny\AppData\Local\npm-cache\_logs\2023-11-26T12_06_26_509Z-debug-0.log
```

It's not clear to me if the download is failing (that URL appears correct) or if `chmod` is failing to find `src.exe`. There's not much more detail in the full log:

[2023-11-26T12_06_26_509Z-debug-0.log](https://github.com/sourcegraph/src-cli/files/13466696/2023-11-26T12_06_26_509Z-debug-0.log)

I'm using Windows 11:

![image](https://github.com/sourcegraph/src-cli/assets/1078012/b783f1be-3f8e-4914-8dd2-e80621752079)

With these versions of node/npm:

```
PS C:\Dev\Sourcegraph\cody> node --version
v18.18.2
PS C:\Dev\Sourcegraph\cody> npm --version
10.1.0
```

--- a/npm-distribution/.gitignore
+++ b/npm-distribution/.gitignore
@@ -1,3 +1,4 @@
 node_modules/
 src
 LICENSE
+README.md

--- a/npm-distribution/copy-files.js
+++ b/npm-distribution/copy-files.js
@@ -1,0 +1,8 @@
+const fs = require('fs')
+const path = require('path')
+
+const licensePath = path.join(__dirname, '..', 'LICENSE')
+const readmePath = path.join(__dirname, '..', 'README.md')
+
+fs.copyFileSync(licensePath, path.join(__dirname, 'LICENSE'))
+fs.copyFileSync(readmePath, path.join(__dirname, 'README.md'))

--- a/npm-distribution/install.js
+++ b/npm-distribution/install.js
@@ -56,9 +56,12 @@ get(assetURL, (response) => {
   }
   response
     .pipe(zlib.createGunzip())
-    .pipe(tar.x({ filter: (x) => x === "src" }))
-    .addListener("close", () => fs.chmodSync(executableName, "755"));
-});
+    .pipe(tar.x({ filter: (x) => x === executableName }))
+    .addListener("close", () => {
+      if (process.platform !== 'win32') {
+        fs.chmodSync(executableName, "755");
+      }
+    });});
 
 // Follow redirects.
 function get(url, callback) {

--- a/npm-distribution/package.json
+++ b/npm-distribution/package.json
@@ -7,7 +7,7 @@
   "license": "Apache-2.0",
   "scripts": {
     "install": "node install.js",
-    "prepack": "cp ../LICENSE . && cp ../README.md ."
+    "prepack": "node copy-files.js"
   },
   "main": "src.js",
   "bin": {

